e2e/nextjs/src/app/components/highlight-identify.tsx

```diff
@@ -1,14 +1,24 @@
 'use client'
 
 import { H } from '@highlight-run/next/client'
+import { initialize } from '@launchdarkly/js-client-sdk'
 import { useEffect } from 'react'
 
+H.init('1', {
+	debug: { clientInteractions: true, domRecording: true },
+})
+const ldClient = initialize(process.env.NEXT_PUBLIC_LAUNCHDARKLY_SDK_KEY)
+// @ts-ignore
```

Comment:
We may want to add a comment explaining the ignore, or add a global type definition.

Reply:
Or, if we do, then figure out the limitations and an explanation of how to format a context.
Similar to our guides for how to use an OpenFeature context: https://github.com/launchdarkly/openfeature-java-server/?tab=readme-ov-file#examples

Reply:
Looked into this more with next.js client side rendering and found a different solution with a lazy initialization in the component. Could be pulled out into a shared context, but for the test app going to leave this a bit hacky.